edx_exams/apps/api/v1/urls.py

```diff
@@ -32,4 +33,7 @@
     path('exams/attempt',
          ExamAttemptView.as_view(),
          name='exams-attempt',),
+    re_path(fr'exams/attempt/course_id/{COURSE_ID_PATTERN}/content_id/{CONTENT_ID_PATTERN}',
```

Comment:
In the edx-proctoring url path, the content_id was a query parameter. The MFE always appends the parameter so I decided to make it a path parameter for edx-exams